src/server.js

```diff
@@ -35,9 +35,18 @@ module.exports = function() {
 		log.warn("Server is public and set to use LDAP. Set to private mode if trying to use LDAP authentication.");
 	}
 
+	var host = config.host || null;
+	var port = config.port;
+
+
+	if (typeof config.host !== "undefined" && config.host.includes("unix:")) {
+		host = null;
+		port = config.host.replace("unix:", "");
```

Comment:
I'd use `substr` or a `/^unix:/` replace. On Linux the only invalid character in a path is `/`, so `unix:/run/unix:sockets/lounge.sock` is a perfectly valid path and would fail.